src/Glpi/Form/QuestionType/AbstractQuestionTypeSelectable.php

```diff
@@ -92,7 +93,7 @@ protected function getFormInlineScript(): string
     #[Override]
     public function loadJavascriptFiles(): array
     {
-        return ['js/form_question_selectable.js'];
+        return [];
```

Comment:
I guess we can delete the `loadJavascriptFiles` method then since it is no longer used.
One less method for the `QuestionType` interface is good to take at it very crowded.

I've added it to the backlog, can be done in another PR.

Reply:
I originally left it in because I wasn't sure how plugin support would work, but I guess they don't need to load JS from this method.